bundle.Dockerfile

```diff
@@ -6,7 +6,7 @@ LABEL operators.operatorframework.io.bundle.manifests.v1=manifests/
 LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
 LABEL operators.operatorframework.io.bundle.package.v1=kaoto-operator
 LABEL operators.operatorframework.io.bundle.channels.v1=alpha
-LABEL operators.operatorframework.io.metrics.builder=operator-sdk-v1.15.0+git
+LABEL operators.operatorframework.io.metrics.builder=operator-sdk-v1.30.0
```

Comment:
The commit of both this file and the bundle directory is maybe unnecessary as they are generated as product artefacts. We've found that automated pipelines (or manual release processes) should be responsible for running the make rules then releasing the results. If committed then files are in danger of becoming out-of-date (if the commands have not been re-run prior to changes in the Makefile etc.) or causing conflicts (migrating changes from upstream to product repositories).

Reply:
I do agree, at this stage I've committed them just because that is what I found on the repo but should not be included

Reply:
removed